server/controllers/reviewController/reviewController.js

```diff
@@ -0,0 +1,48 @@
+const router = require('express').Router();
+const ReviewRepository = require('./reviewRepository');
+const { noReviews, serverError } = require('../../constants/httpResponses');
+
+router.get('/:workspaceId', async (req, res) => {
+    const repo = new ReviewRepository(req.params.workspaceId);
+    try {
+        const reviews = await repo.get();
+        if (reviews.length === 0) return res.status(404).json({ message: noReviews });
+
+        return res.status(200).json({ reviews });
+    } catch (error) {
+        return res.status(500).json({ message: serverError });
+    }
+});
+
+router.post('/:workspaceId', async (req, res) => {
+    const repo = new ReviewRepository(req.params.workspaceId);
+    try {
+        const newReview = await repo.create(req.body);
+        return res.status(201).json(newReview);
```

Comment:
201 is more specific, but less common. I believe every production API I have worked with returns 200 for successful post requests. It's also more user friendly for people new to APIs